cli/lib/kontena/plugin_manager.rb

```diff
@@ -56,6 +56,8 @@ def uninstall_plugin(plugin_name)
     # Search rubygems for kontena plugins
     # @param pattern [String] optional search pattern
     def search_plugins(pattern = nil)
+      require 'excon'
+      require 'json'
```

Comment:
Another option is autoload. 

Plugin manager is a special case because it's loaded for every command so if it requires outside of methods it's global and those two are quite slow to load. 

Of course the root cause is that these methods should be in classes of their own.